components/ActionSheet/BUILD

```diff
@@ -51,11 +53,11 @@ mdc_objc_library(
     hdrs = native.glob(["src/TypographyThemer/*.h"]),
     includes = ["src/TypographyThemer"],
     sdk_frameworks = ["UIKit"],
+    visibility = ["//visibility:public"],
```

Comment:
Tip: Next time please put `buildifier` changes in a precursor PR. That will help reduce "noise" for reviewers.